Resources/Prototypes/Entities/Objects/Power/powercells.yml

```
@@ -296,6 +296,7 @@
     walkModifier: 0.8
     sprintModifier: 0.8
   - type: HeldSpeedModifier
+  - type: EmpImmune # Мехи теперь имеют прямой урон от ЭМИ.
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)